esp/services/ws_esdlconfig/ws_esdlconfigservice.cpp

```diff
@@ -695,6 +695,31 @@ bool CWsESDLConfigEx::onPublishESDLBinding(IEspContext &context, IEspPublishESDL
     return true;
 }
 
+void setContainerizedEspBindingNameByPort(StringBuffer& espPort, StringBuffer& espBindingName)
+{
+    Owned<IPropertyTree> globalConfig = getGlobalConfigSP();
+    if (!globalConfig)
+        throw makeStringExceptionV(-1, "Can't use global config to find any esp binding for port %s", espPort.str());
+
+    VStringBuffer xpath("services[@class='esp'][@port='%s']", espPort.str());
+    IPropertyTree* process = globalConfig->queryPropTree(xpath);
```

Comment:
You can just use getGlobalConfigSP() directly here and not declare it above.

IPropertyTree* process = getGlobalConfigSP()->queryPropTree(xpath);

Reply:
fixed